ui/App/ProfileScreen.svelte

```diff
@@ -70,15 +185,73 @@
     </Button>
   </Header>
 
-  <ActionBar>
-    <TabBar slot="left" tabs={tabs(activeTab)} />
-  </ActionBar>
-
-  {#if activeTab === "projects"}
-    <ProjectsTab />
-  {:else if activeTab === "following"}
-    <FollowingTab />
-  {:else}
-    {unreachable(activeTab)}
+  {#if $profileProjectsStore.status === remote.Status.Success}
+    {#if (Array.isArray($profileProjectsStore.data) && $profileProjectsStore.data.length === 0) || $profileProjectsStore.data === null}
+      <EmptyState
+        text="You don’t have any projects yet."
+        primaryActionText="Start your first project"
+        on:primaryAction={create} />
```

Comment:
`create` is a tiny function so it makes sense to inline it to reduce indirection.

```suggestion
        on:primaryAction={() => modal.toggle(CreateProjectModal); } />
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2398/commits/485ac38bc0f582ae01cf121375c22e1245297692